quota/noop.go

```diff
@@ -17,10 +17,23 @@ package quota
 import (
 	"context"
 	"fmt"
+
+	"github.com/golang/glog"
 )
 
+// noopManagerName represents the noop quota implementation.
+const noopManagerName = "noop"
```

Comment:
Could you make sure `storage` also uses "manager name" rather than "provider name"?

Reply:
It doesn't have an equivalent constant - it just uses the strings inline when calling `storage.RegisterProvider()`.